src/_data/starters/nonplain-11ty-starter.json

```diff
@@ -0,0 +1,7 @@
+{
+	"url": "https://github.com/nonplain/nonplain-11ty-starter#readme",
+	"name": "nonplain-11ty-starter",
+	"description": "Create an 11ty website out of nonplain files",
+	"demo": "https://nonplain-11ty-starter.vercel.app/",
+  "author": "jaredgorski"
```

Comment:
Q: Is this your Twitter username?

Reply:
No; I don't have a Twitter. I'll remove this then